app/views/layouts/application.html.haml

```diff
@@ -2,11 +2,13 @@
 %html
   %head
     %meta{content: "text/html; charset=UTF-8", http: {equiv: "Content-Type"}}
+    %link{:href => "https://fonts.googleapis.com/css?family=Limelight", :rel => "stylesheet"}
```

Comment:
Please do not use old hash syntax (I will not comment on this anymore, just silently reject PR's that contain this code)